libs/angular-components/community/components/form/file-dropzone/file-dropzone.component.html

```diff
@@ -0,0 +1,74 @@
+<input
+  #fileInput
+  class="tedi-file-dropzone__input"
+  type="file"
+  [multiple]="multiple()"
+  [id]="inputId()"
+  [attr.webkitdirectory]="uploadFolder() ? '' : null"
+  [accept]="accept()"
+  [multiple]="multiple()"
+  (change)="selectionChange($event)"
+/>
+<button [className]="classes()" (click)="onContainerClick()">
+  <div class="tedi-file-dropzone__label-wrapper">
+    <tedi-icon
+      [name]="isDragActive() ? 'file_upload' : 'attach_file'"
+      [color]="disabled() ? 'tertiary' : 'secondary'"
+    />
+    <label tedi-label class="tedi-file-dropzone__label">{{
+      isDragActive()
```

Comment:
very small thing, but this could be just  (isDragActive() ? "file-upload.drag-and-drop" : label()) | tediTranslate